scratch/xbgpu/run-xbgpu.sh

```diff
@@ -13,19 +13,29 @@ spectra_per_heap=${spectra_per_heap:-256}
 samples_between_spectra=${samples_between_spectra:-$((channels*2))}
 heap_accumulation_threshold=$(python -c "print(round($int_time * $adc_sample_rate / $samples_between_spectra / $spectra_per_heap))")
 
+index="$1"
 nproc=$(nproc)
-step=$(($nproc / 4))
-affinity="$(($1 * step))"
+step=$((nproc / 4))
+affinity="$((index * step))"
 rx_affinity=$affinity
 rx_comp=$rx_affinity
-tx_affinity=$(($affinity + 1))
+tx_affinity=$((affinity + 1))
```

Comment:
Ha, the more you know.

Reply:
Yeah, it turns out in `$(( ... ))` you don't need to prefix the variables with `$` (although it still works).